cunumeric/random/random.py

```diff
@@ -43,6 +50,50 @@ def beta(
     size: Union[NdShapeLike, None] = None,
     dtype: npt.DTypeLike = np.float64,
```

Comment:
`dtype` is not documented applies to other functions in this file as well.

Reply:
`dtype` will be removed in a follow-on PR as well. (For some reason, the first PR deviated from the NumPy API for no reason.)